builtin/logical/pki/path_manage_issuers.go

```diff
@@ -354,6 +451,88 @@ func pathRevokeIssuer(b *backend) *framework.Path {
 		Operations: map[logical.Operation]framework.OperationHandler{
 			logical.UpdateOperation: &framework.PathOperation{
 				Callback: b.pathRevokeIssuer,
+				Responses: map[int][]framework.Response{
+					http.StatusOK: {{
+						Description: "OK",
+						Fields: map[string]*framework.FieldSchema{
+							"issuer_id": {
+								Type:        framework.TypeString,
+								Description: `ID of the issuer`,
+								Required: true,
+							},
+							"issuer_name": {
+								Type:        framework.TypeString,
+								Description: `Name of the issuer`,
+								Required: true,
+							},
+							"key_id": {
+								Type:        framework.TypeString,
+								Description: `ID of the Key`,
+								Required: true,
+							},
+							"certificate": {
+								Type:        framework.TypeString,
+								Description: ``,
+								Required: true,
+							},
+							"manual_chain": {
+								Type:        framework.TypeCommaStringSlice,
+								Description: ``,
+								Required: true,
+							},
+							"ca_chain": {
+								Type:        framework.TypeCommaStringSlice,
+								Description: ``,
+								Required: true,
+							},
+							"leaf_not_after_behavior": {
+								Type:        framework.TypeString,
+								Description: ``,
+								Required: true,
+							},
+							"usage": {
+								Type:        framework.TypeCommaStringSlice,
```

Comment:
```suggestion
								Type:        framework.TypeString,
```
looks like this is a string

Reply:
hmmm. Let me double check. on the docs it looked like it could be a list

Reply:
`"usage": "read-only,issuing-certificates,crl-signing"`. I guess it looks like this. So, I guess just a string then.